openapi.yaml

```diff
@@ -16006,14 +16032,24 @@ components:
           example: 12
         expiry_year:
           type: integer
-          description: >
+          minLength: 4
+          maxLength: 4
+          description: |
             A four-digit integer representing the expiration year of
             your credit card.
 
             The combination of `expiry_month` and `expiry_year`
             must result in a month/year combination of the current month or in
             the future. An expiration date set in the past is invalid.
           example: 2020
+        cvv:
+          type: string
+          minLength: 3
+          maxLength: 4
+          format: digits
+          description: >
+            CVV (Card Verification Value) of the credit card, typically found on the back of the card. **** 
```

Comment:
What are the four asterisks here for?

Reply:
A placeholder that I forgot to remove! Removing now...